src/PathExpressionHandler.js

```diff
@@ -9,10 +9,10 @@ export default class PathExpressionHandler {
     // Add all predicates to the path
     while (current.parent) {
       // Obtain and store predicate
-      if (!current.predicate)
-        throw new Error(`Expected predicate in ${current}`);
```

Comment:
I see in the tests that no error is thrown anymore for this case. Why is that?

Reply:
I removed this because things like `add()` and `remove()` will add `mutationExpressions` to the path instead of a `predicate`. This means that not all path segments will have `predicates` anymore. So I changed the code to just ignore all segments that don't have a `predicate`.

As far as I can see, there is no way for users to trigger such an invalid path segment to be created, so that an error like this _must_ be thrown, right?

Reply:
Got it, thanks.